SPECS/cryptsetup/cryptsetup.spec

```diff
@@ -1,135 +1,146 @@
 disk verification using dm-verity kernel module.
- 
+
 %package -n integritysetup
-Group: Applications/System
-Summary: A utility for setting up dm-integrity volumes
-Requires: cryptsetup-libs = %{version}-%{release}
- 
+Summary:        A utility for setting up dm-integrity volumes
+Group:          Applications/System
+Requires:       cryptsetup-libs = %{version}-%{release}
+
 %description -n integritysetup
 The integritysetup package contains a utility for setting up
 disk integrity protection using dm-integrity kernel module.
- 
+
 %package reencrypt
-Group: Applications/System
-Summary: A utility for offline reencryption of LUKS encrypted disks.
-Requires: cryptsetup-libs = %{version}-%{release}
- 
+Summary:        A utility for offline reencryption of LUKS encrypted disks.
+Group:          Applications/System
+Requires:       cryptsetup-libs = %{version}-%{release}
+
 %description reencrypt
 This package contains cryptsetup-reencrypt utility which
 can be used for offline reencryption of disk in situ.
- 
+
 %prep
 %setup -q -n cryptsetup-%{version}
 %patch0 -p1
 chmod -x misc/dracut_90reencrypt/*
- 
+
 %build
-%configure --enable-fips --enable-pwquality --enable-internal-sse-argon2 --with-crypto_backend=openssl --with-default-luks-format=LUKS2
+./autogen.sh
+%configure --enable-fips --enable-pwquality --enable-internal-sse-argon2 --with-default-luks-format=LUKS2 --disable-ssh-token
 make %{?_smp_mflags}
- 
+
 %install
 make install DESTDIR=%{buildroot}
-rm -rf %{buildroot}/%{_libdir}/*.la
- 
+find %{buildroot} -type f -name "*.la" -delete -print
+
 %find_lang cryptsetup
- 
+
 %post -n cryptsetup-libs -p /sbin/ldconfig
- 
 %postun -n cryptsetup-libs -p /sbin/ldconfig
- 
+
 %files
 %{!?_licensedir:%global license %%doc}
 %license COPYING
 %doc AUTHORS FAQ docs/*ReleaseNotes
 %{_mandir}/man8/cryptsetup.8.gz
 %{_sbindir}/cryptsetup
- 
+
 %files -n veritysetup
 %{!?_licensedir:%global license %%doc}
 %license COPYING
 %{_mandir}/man8/veritysetup.8.gz
 %{_sbindir}/veritysetup
- 
+
 %files -n integritysetup
 %{!?_licensedir:%global license %%doc}
 %license COPYING
 %{_mandir}/man8/integritysetup.8.gz
 %{_sbindir}/integritysetup
- 
+
 %files reencrypt
 %{!?_licensedir:%global license %%doc}
 %license COPYING
 %doc misc/dracut_90reencrypt
 %{_mandir}/man8/cryptsetup-reencrypt.8.gz
 %{_sbindir}/cryptsetup-reencrypt
- 
+
 %files devel
 %doc docs/examples/*
 %{_includedir}/libcryptsetup.h
 %{_libdir}/libcryptsetup.so
 %{_libdir}/pkgconfig/libcryptsetup.pc
- 
+
 %files libs -f cryptsetup.lang
 %{!?_licensedir:%global license %%doc}
 %license COPYING COPYING.LGPL
 %{_libdir}/libcryptsetup.so.*
 %{_tmpfilesdir}/cryptsetup.conf
 %ghost %dir /run/cryptsetup
- 
+
 %changelog
-* Thu Dec 16 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.3.3-3
-- Removing the explicit %%clean stage.
+* Tue Jan 18 2022 Daniel McIlvaney <damcilva@microsoft.com> - 2.4.3-1
+- Update to version 2.4.3
+- We currently do not support SSH tokens, re-enable once libssh is available
```

Comment:
Should we create a bug to track it if not already created.

Reply:
Not a bad idea, I'll touch base with Jon and do so.

Reply:
Done